extensions/indexes/range/src/main/java/org/exist/indexing/range/ComplexRangeIndexConfigElement.java

```diff
@@ -63,15 +63,16 @@ public ComplexRangeIndexConfigElement(final Element node, final NodeList childre
         for (int i = 0; i < children.getLength(); i++) {
```

Comment:
I wonder if this could benefit from a nicer iteration as well
```java
for (child : children) {}
```
I would also like to reverse the condition at the beginning to `if (child.getNodeType() != Node.ELEMENT_NODE) { return; }`